examples/ui/grid.rs

```diff
@@ -0,0 +1,161 @@
+//! Demonstrates how CSS Grid layout can be used to lay items out in a 2D grid
+use bevy::prelude::*;
+
+const ALIGN_ITEMS_COLOR: Color = Color::rgb(1., 0.066, 0.349);
+const MARGIN: Val = Val::Px(5.);
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins.set(WindowPlugin {
+            primary_window: Some(Window {
+                resolution: [800., 600.].into(),
+                title: "Bevy CSS Grid Layout Example".to_string(),
+                ..default()
+            }),
+            ..default()
+        }))
+        .add_startup_system(spawn_layout)
+        .run();
+}
+
+fn rect(color: Color) -> NodeBundle {
+    NodeBundle {
+        background_color: BackgroundColor(color),
+        ..default()
+    }
+}
+
+fn spawn_layout(mut commands: Commands, asset_server: Res<AssetServer>) {
+    let font = asset_server.load("fonts/FiraSans-Bold.ttf");
+    commands.spawn(Camera2dBundle::default());
+    commands
+        .spawn(NodeBundle {
+            style: Style {
+                display: Display::Grid,
+                size: Size::all(Val::Percent(100.)),
+                grid_template_columns: vec![GridTrack::flex(1.), GridTrack::flex(2.), GridTrack::flex(1.)],
+                grid_template_rows: vec![GridTrack::auto(), GridTrack::px(150.), GridTrack::flex(1.)],
+                gap: Size::all(Val::Px(20.)),
```

Comment:
```suggestion
                // Setting Display to Grid will enable the grid layout algorithm to be used.
                display: Display::Grid,
                // Make sure the node fills the entire window
                size: Size::all(Val::Percent(100.)),
                // FIXME: some comment explaining what this line does
                grid_template_columns: vec![GridTrack::flex(1.), GridTrack::flex(2.), GridTrack::flex(1.)],
                // FIXME: some comment explaining what this line does
                grid_template_rows: vec![GridTrack::auto(), GridTrack::px(150.), GridTrack::flex(1.)],
                // FIXME: some comment explaining what this line does
                gap: Size::all(Val::Px(20.)),
```
Since this is probably one of the more important section of this example, it would be great to have comments on these lines in particular :)